sixtools/rixs_wrapper.py

```diff
@@ -0,0 +1,221 @@
+import numpy as np
+from scipy.interpolate import interp1d
+from pims import pipeline
+from rixs.process2d import apply_curvature, image_to_photon_events
+
+# Eventually we will create this information from the configuration
+# attributes in ophyd.
+process_dict_low_2theta = {'light_ROI': [slice(175, 1609), slice(1, 1751)],
+                           'curvature': np.array([0., 0., 0.]),
+                           'bins': None}
+
+process_dict_high_2theta = {'light_ROI': [slice(175, 1609), slice(1753, 3503)],
+                            'curvature': np.array([0., 0., 0.]),
+                            'bins': None}
+
+
+process_dicts = {'low_2theta': process_dict_low_2theta,
+                 'high_2theta': process_dict_high_2theta}
+
+
+@pipeline
```

Comment:
This also appears to include all of the changes from the other PR, is this true? if so it makes it hard to review. Can we remove any changes that are in the other merged PR?

Reply:
Yes, I think this is true. I'm not 100% sure how to go about this. I guess if we close this and I open a new pull request it will diff w.r.t. the branch that now includes the other changes?